package.json

```diff
@@ -23,7 +23,7 @@
   "dependencies": {
     "commander": "2.16.0",
     "diff": "3.5.0",
-    "eslint": "5.16.0",
+    "eslint": "8.28.0",
```

Comment:
Let's also update the `peerDependencies` and `engines`.

One thought would be to use [what is current in the ESLint generator](https://github.com/eslint/generator-eslint/blob/v5.0.0/plugin/templates/_package.json#L31-L36).

```
"eslint": ">=7"
"node": "^14.17.0 || ^16.0.0 || >= 18.0.0"
```

Another thought would be to use [what is declared in ESLint version 8.28.0](https://github.com/eslint/eslint/blob/v8.28.0/package.json#L168).

```
"eslint": ">=8.28.0"
"node": "^12.22.0 || ^14.17.0 || >=16.0.0"
```

I mostly care about ensuring node8 isn't supported because I think the trimStart() usage was always a problem for it and preventing the install would have been more obvious about it.